src/sickle.ts

```diff
@@ -68,7 +70,10 @@ export function getJSDocAnnotation(comment: string): JSDocComment {
     let match = line.match(/^@(\S+) *(.*)/);
     if (match) {
       let [_, tagName, text] = match;
-      if (text[0] == '{') {
+      if (tagName == 'type') {
+        throw new Error('@type annotations are not allowed');
```

Comment:
There are a few shortcuts for type annotations, for example @const {string}. Can handle them in a separate cl.